app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
       android:name="com.google.android.geo.API_KEY"
       android:value="${MAPS_API_KEY}" />
 
-      <activity android:name=".MainActivity">
+      <activity android:name=".MainActivity" android:exported="true">
```

Comment:
we now need to explicitly specify this activities with an intent filter as exported. the main launcher activity should always be exported, so other apps (and android studio!) can launch it